dydx/dydxViews/dydxViews/_v4/Rating/dydxRateAppView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  dydxRateAppView.swift
+//  dydxUI
+//
+//  Created by Michael Maguire on 2/28/24.
+//  Copyright © 2024 dYdX Trading Inc. All rights reserved.
+//
+
+import SwiftUI
+import PlatformUI
+
+// Define a UIViewRepresentable to use UIVisualEffectView in SwiftUI
+struct BlurView: UIViewRepresentable {
+    var style: UIBlurEffect.Style
+
+    func makeUIView(context: Context) -> UIVisualEffectView {
+        return UIVisualEffectView(effect: UIBlurEffect(style: style))
+    }
+
+    func updateUIView(_ uiView: UIVisualEffectView, context: Context) {
+        uiView.effect = UIBlurEffect(style: style)
+    }
+}
+
+public class dydxRateAppViewModel: PlatformViewModel {
+    @Published public var positiveRatingIntentAction: (() -> Void)?
+    @Published public var negativeRatingIntentAction: (() -> Void)?
+    @Published public var deferAction: (() -> Void)?
+
+    public static var previewValue: dydxRateAppViewModel = {
+        let vm = dydxRateAppViewModel()
+        return vm
+    }()
+
+    private func createButtonContent(title: String) -> AnyView {
+        HStack {
+            Spacer()
+            Text(title)
+                .themeFont(fontSize: .medium)
+                .themeColor(foreground: .textPrimary)
```

Comment:
If the button background is purple, it looks better when the text remains white regardless of theme.

Reply:
the design here is not finalized. I will be following up with Dan to finalize a better looking design for the pop-up itself.